modules/virt-verify-status-bootsource-update.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-verify-status-bootsource-update_{context}"]
+= Verify status of automatic updating for a boot source
+
+You can verify the status of a boot source to confirm that automatic importing and updating is enabled or disabled.
+
+The `status` section of each boot source listed in the `status.dataImportChronTemplates` field of the `HyperConverged` CR indicates if automatic importing and updating is enabled or disabled. For example, `commonTemplate: true` indicates that automatic importing and updating has been enabled for a system-defined (`commonTemplate`) boot source.
```

Comment:
The status field of a boot source **_does not_** indicate anything about enabling or disabling. If the boot source  is listed, then it's enabled. That it.

The  status tells things about the specofic boot  source. Or to br more specific, if it's a system defined or user defined boot source.

Reply:
OK, I stripped this module of anything relating to enablement/disablement of auto updating.